grails-app/assets/javascripts/PaymentReceiptController.js

```diff
@@ -0,0 +1,32 @@
+function PaymentReceiptController(reference) {
+
+    const init = () => {
+        bindDownloadButton();
+    }
+
```

Comment:
pq mudou a forma de usar os métodos aqui?
```suggestion
    function init() {
        bindDownloadButton();
    }

```